README.md

```diff
@@ -10,18 +10,15 @@
 
 # WeatherBench 2 - A benchmark for the next generation of data-driven global weather models
 
-
-```
-[TODO] Paper citation
-```
+[Paper](https://drive.google.com/file/d/1OqSR5H_h2y2HXEf4LVFO7ZLxgdafK2Va/view?usp=sharing)
 
 ## Why WeatherBench?
 
 WeatherBench 2 is a framework for evaluating and comparing data-driven and traditional numerical weather forecasting models. WeatherBench consists of:
 - Publicly available, cloud-optimized ground truth and baseline datasets. For a complete list, see [this page](https://weatherbench2.readthedocs.io/en/latest/data-guide.html). 
 - Open-source evaluation code. See this [quick-start](https://weatherbench2.readthedocs.io/en/latest/evaluation.html) to explore the basic functionality or the [API docs](https://weatherbench2.readthedocs.io/en/latest/api.html) for more detail. Since high-resolution forecast files can be large, the WeatherBench 2 code was written with scalability in mind. See the [command-line scripts](https://weatherbench2.readthedocs.io/en/latest/command-line-scripts.html) based on [Xarray-Beam](https://xarray-beam.readthedocs.io/en/latest/) and [this guide](https://weatherbench2.readthedocs.io/en/latest/beam-in-the-cloud.html) for running the scripts on GCP using [DataFlow](https://cloud.google.com/dataflow).
-- A [website] displaying up-to-date scores of many of the state-of-the-art data-driven and physical approaches.
-- A [paper] describing the rationale behind the evaluation setup.
+- A [website](https://weatherbench.research.google.com) displaying up-to-date scores of many of the state-of-the-art data-driven and physical approaches.
```

Comment:
This site cannot be reached.

Reply:
The website is not live yet. This was actually the wrong link. It will be live at: https://sites.reasearch.google/weatherbench on Thursday

Reply:
Actually, let's wait until Thursday morning with this PR. I will then also update the link for the arxiv paper.

Reply:
SGTM! Thursday at what time (and what timezone) will we hit the publish button? :)

Reply:
+1 to an arxiv link.

Reply:
The exact timing is still tbd